Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apk add --update --virtual build-dependencies build-base linux-headers && \
     make build
 
 FROM alpine
+RUN apk add hwdata-pci
```

Comment:
Where will the package be installed, `/usr/share/misc/pci.ids/`?

Reply:
> Where will the package be installed, `/usr/share/misc/pci.ids/`?

It will be installed at `/usr/share/hwdata/pci.ids`